app/lib/hyrax/hirmeos/metrics_tracker.rb

```diff
@@ -6,10 +6,24 @@ def client
     @client ||= Hyrax::Hirmeos::Client.new(username, password, metrics_base_url, translation_base_url, secret)
   end
 
-  def submit_to_hirmeos(resource_id, resource_json)
-    response = client.get_resource(resource_id)
+  def submit_work_to_hirmeos(work_id, work_json)
+    response = client.get_resource(work_id)
     return if response.success?
-    client.post_resource(resource_json)
+    client.post_resource(work_json)
+  end
+
+  def submit_file_set_to_hirmeos(file_set_id, file_set_json)
+    response = client.get_resource(file_set_id)
+    json_response = JSON.parse(response.body)
+    return if file_already_registered?(json_response)
```

Comment:
Its best to leave spaces around a return/guard statement so that its obvious what is happening